main.c

```diff
@@ -44,7 +45,7 @@
 1: Development one
 
 */
-#define FW_VARIANT 1
+#define FW_VARIANT 0
```

Comment:
Please keep it as 1 for the git tree as it should be a development release by default.